### DISCUSSION TITLE
米哈游

### DISCUSSION THREAD

**Discussion by Li-Yuchen-cn** [share121/inter-knot#28] [其他]
![mihoyo](https://github.com/user-attachments/assets/1a402a2d-2d91-4c88-82c4-3b759fc49f95)
米哈游（全称上海米哈游网络科技股份有限公司，简称米哈游或miHoYo，中国大陆以外的英文名称为HoYoverse）是一家成立于2012年2月13日，总部位于中国上海的电子游戏公司。在游戏之外，米哈游的文化产品还有动画系列、小说、漫画、音乐等。
米哈游以制作动作游戏《崩坏》系列及 《原神》和《绝区零》闻名，其中发布于2020年的开放世界动作角色扮演游戏《原神》是史上营收最高的手机游戏之一。米哈游于2021年被中华人民共和国商务部列为2021-2022年度国家文化出口重点企业。
目前，米哈游已在上海、北京、香港、台北、新加坡、美国洛杉矶、加拿大蒙特利尔、日本东京、韩国首尔等地建立多个工作室、子公司/关系企业。目前共有员工4000余人。在中国大陆，米哈游主要子公司包括：上海米哈游影铁科技有限公司（运营《崩坏学园2》、《崩坏3》、《原神》、《绝区零》）、上海米哈游天命科技有限公司（运营《未定事件簿》）、上海米哈游海渊城科技有限公司（运营《崩坏．星穹铁道》）、上海米哈游璃月科技有限公司、米哈游（济南）有限公司（其一并管理米哈游-山东大学联合研究院）等。在海外，自2022年2月15日起，米哈游通过在新加坡设立的间接子公司兼海外代理人Cognosphere Pte., Ltd.（识隙之城私人有限公司）创立HoYoverse品牌，代替本部在中国大陆以外的发行业务，由蔡浩宇担任董事长。2023年9月蔡浩宇退出米哈游董事会，董事长由刘伟接任。

**Comment by asdhvq** [comment]
谁问你了

**Comment by Luli-Chen** [comment]
这个委托接不了

**Comment by koyki233** [comment]
这个委托我不敢接
